apps/storybook/src/unstable/datacom-feature-highlight.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { StoryFn } from '@storybook/react';
+import {
+  DatacomFeatureHighlight,
+  DatacomFeatureHighlightGroup,
+} from '@datacom/endeavour-react';
+
+type FeatureHighlightProps = React.ComponentProps<
+  typeof DatacomFeatureHighlight
+>;
+
+export default {
+  title: 'Feature Highlight',
+  component: DatacomFeatureHighlight,
+  argTypes: {
+    icon: {
+      name: 'Variant',
+      description: 'Display image or icon from a set of pre-defined images',
+      control: {
+        type: 'select',
+        labels: {
+          globe: 'Icon',
+          '': 'Image',
+        },
+      },
+      options: ['', 'globe'],
+      defaultValue: '',
+    },
+    featureTitle: {
+      name: 'Title',
+      defaultValue: 'Title',
+      description: 'Text of title displayed.',
+      type: { name: 'string', required: 'true' },
+    },
+    description: {
+      name: 'Description',
+      defaultValue:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore etdolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+      description: 'Description of display feature',
+      type: { name: 'string' },
+    },
+    ctaText: {
+      name: 'CTA Label',
+      defaultValue: 'Learn more',
+      type: { name: 'string' },
+    },
+    url: {
+      name: 'CTA URL',
+      defaultValue: 'https://datacom.com',
+      description: 'URL that  should link to.',
+      type: { name: 'string' },
+    },
+  },
+};
+let key = 0;
+let previousDesc = '';
+
+const Template: StoryFn<FeatureHighlightProps & { description: string }> = (
+  args
+) => {
+  const { description, ctaText, featureTitle, url, icon } = args;
+  if (description !== previousDesc) {
+    previousDesc = description;
+    key++;
+  }
+  return (
+    <div style={{ maxWidth: 386 }}>
+      <DatacomFeatureHighlight
+        key={key}
+        imageUrl="https://images.pexels.com/photos/15638791/pexels-photo-15638791.jpeg"
+        featureTitle={featureTitle}
+        icon={icon}
+        url={url}
+        ctaText={ctaText}>
+        {description && <span slot="description">{description}</span>}
+      </DatacomFeatureHighlight>
+    </div>
+  );
+};
+
+export const FeatureHighlight = Template.bind({});
+FeatureHighlight.args = {};
+
+export const FeatureHighlightGroup = () => {
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center', height: 600 }}>
+      <div style={{ maxWidth: 1254, width: '100%' }}>
+        <DatacomFeatureHighlightGroup>
```

Comment:
Just thought of this, so sorry for the extra work. It might be good to make a story where you can change the itemsPerRow value to be either 3 or 4. Would be good to show that off, as there are examples of both in the designs. You might need to move the group into another story file to do it, but I think it's worth it to show off.

Reply:
No problem, I have added the new group story with ItemsPerRow control without removing another group story. The reason is that the designer wants it as it is along with the new one in the showcase. Please, review again when you can.

Reply:
Sally just added some additional feedback that's related to the new story added. Best to address those changes, and then I can re-review this once more.

Reply:
Thanks, Angel!  After the designers' feedback, I updated the stories and changed the files' names for sorting purposes. Please review again when you can.